core/src/main/scala/kafka/tools/EndToEndLatency.scala

```diff
@@ -89,9 +91,9 @@ object EndToEndLatency {
     }
 
     //Ensure we are at latest offset. seekToEnd evaluates lazily, that is to say actually performs the seek only when
-    //a poll() or position() request is issued. Hence we need to poll after we seek to ensure we see our first write.
+    //a position() request is issued. Hence we need to position after we seek to ensure we see our first write.
     consumer.seekToEnd(Collections.emptyList())
-    consumer.poll(0)
+    consumer.assignment().asScala.foreach(consumer.position)
```

Comment:
As far as I can tell, the assignment will be empty here. I don't think the old code worked either as intended because it did the `seekToEnd` prior to the call to `poll()`. For an end-to-end latency test, I actually wonder if we should just use manual assignment and take the group management out of the picture?

Reply:
Will change to manual assignment, something like this (which seems to work).
```
val topicPartitions = consumer.listTopics().asScala
      .flatMap(_._2.asScala.to[List])
      .map(pi => new TopicPartition(pi.topic(), pi.partition()))
      .to[List].asJava
    consumer.assign(topicPartitions)
    consumer.seekToEnd(topicPartitions)
    consumer.assignment().asScala.foreach(consumer.position)
```